packages/react/src/themes/font-awesome/index.ts

```diff
@@ -0,0 +1,8 @@
+import { ThemeInput } from '../types'
+import * as componentStyles from './componentStyles'
+import { default as icons } from './components/Icon/iconNames'
+
+export default {
+  icons,
+  componentStyles,
+} as ThemeInput
```

Comment:
Can we not expose this theme to the users? Everything is breaking if I choose this one on the Icon's docs. I don't see any benefit for the clients to explore this theme..